lib/datapacksjob.js

```diff
@@ -763,10 +763,15 @@ DataPacksJob.prototype.buildManifestFromQueries = async function(jobInfo) {
 
                 for (var query of jobInfo.queries) {
                     if (allTypesForQuery.has(query.VlocityDataPackType)) {
+                        if(query.VlocityDataPackType.includes('Version')){
+                            query.VlocityDataPackType = query.VlocityDataPackType.replace('Version','');
+                        }
                         validQueries.push(query);
                     }
                 }
-
+                if(jobInfo.specificManifestKeys.length === 1 && jobInfo.specificManifestKeys[0].includes('Version')){
```

Comment:
Should also be removed?

Reply:
Updating the manifest keys so as to filter the datapacks correctly after fetching records.